deep_quoridor/src/agents/core/trainable_agent.py

```diff
@@ -203,6 +208,7 @@ def get_action(self, game):
         mask = observation["action_mask"]
 
         if random.random() < self.epsilon:
+            # print(f"rnd-mask: {mask}")
```

Comment:
Maybe we should start using a log framework so that we can have lines like that and turn different levels of logging (we can do it later, no need to do anything now)